packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -202,44 +202,11 @@ const EDiscovery = WebexPlugin.extend({
     // use spread operator to set default options
     options = {...this.config.defaultOptions, ...options};
 
-    return this.request({
-      method: 'GET',
-      service: 'ediscovery',
-      resource: `reports/${reportId}/contents`,
-      qs: {offset: options.offset, size: options.size},
-      timeout: options.timeoutMs
-    });
-  },
-
-  @waitForValue('@')
-  @oneFlight({keyFactory: (reportUrl, options) => createOneFlightKey(reportUrl, options)})
-  /**
-   * Retrieves content associated with a report
-   * @param {String} reportUrl - full destination address (including report id parameter) for the http request being sent
-   * e.g. 'http://ediscovery-intb.wbx2.com/ediscovery/api/v1/reports/3b10e625-2bd5-4efa-b866-58d6c93c505c'
-   * @param {Object} options - optional parameters for this method
-   * @param {number} options.offset - start position from which to retrieve records
-   * @param {number} options.size - the number of records to retrieve
-   * @param {number} options.timeoutMs - connection timeout in milliseconds, defaults to 30s
-   * @returns {Promise<ResponseEntity<[Activity]>>} Http response containing the activities
-   */
-  getRemoteContent(reportUrl, options) {
```

Comment:
Removed in favour of overloading the `getContent(..)` API to support either a UUID or url. Similar approach was used for the other 2 APIs.

The reason for this is explained later on. But it also removes duplication as these methods are essentially equivalent.